terraform/modules/rds/main.tf

```
@@ -0,0 +1,56 @@
+# Non-compliant RDS configuration (missing encryption, backups, and monitoring)
+resource "aws_db_instance" "main" {
+  identifier        = "${var.environment}-db"
```

The RDS instance is not compliant with SOC2 rules due to missing encryption, insufficient backup retention, and lack of monitoring. These are critical for ensuring data security and availability.

```suggestion:-0+0
storage_encrypted = true
backup_retention_period = 30
monitoring_interval = 60
performance_insights_enabled = true
multi_az = true
```